### PR TITLE
88232 Pulling in dropdown text from AEM

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Card dropdown text now comes form AEM and is unique to dashboard cards

### CODE REVIEW

File: graphql/mappers/my-dashboard.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export async function getMyDashboardContent() {
           return {
             id: fragment.scId,
             title: fragment.scTitleEn,
+            dropdownText: fragment.schTasks[0].scLinkTextEn,
```
Comment (DaceyTom2) [inline graphql/mappers/my-dashboard.js:18]: Would it be possible to get this by scid?

File: graphql/mappers/my-dashboard.js [hunk 2]
```diff
@@ -51,6 +52,7 @@ export async function getMyDashboardContent() {
           return {
             id: fragment.scId,
             title: fragment.scTitleFr,
+            dropdownText: fragment.schTasks[0].scLinkTextFr,
```
Comment (DaceyTom2) [inline graphql/mappers/my-dashboard.js:55]: same as above

File: graphql/mappers/my-dashboard.js (hunk 1 above)
Comment (DaceyTom2) [reply graphql/mappers/my-dashboard.js:18]: Approving since IDs won't match per card

Comment (CurtisUnderwood) [reply graphql/mappers/my-dashboard.js:18]: Discussed, will need to change the IDs in AEM

**Review by DaceyTom2 — APPROVED** [review]
Looks good!

**Pull request closed without merging**